### PR TITLE
Tidy-up README and CHANGELOG

### PR DESCRIPTION
**Pull request by thomaseizinger**
Resolves: #186.
Resolves: #200.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,20 @@
 ![Matrix](https://img.shields.io/matrix/xtra-community:matrix.org)
 
 # xtra
-A tiny, fast, and safe actor framework. It is modelled around Actix (copyright and license [here](https://github.com/Restioson/xtra/blob/master/LICENSE-ACTIX)).
 
-For better ergonomics with xtra, try the [spaad](https://crates.io/crates/spaad) crate.
```
Comment (thomaseizinger) [inline README.md:8]: If you want to mention it, please suggest a wording for what it should say @Restioson ! :)

File: README.md [hunk 2]
```diff
@@ -52,31 +51,29 @@ async fn main() {
...
-Check out the [docs](https://docs.rs/xtra) and the [examples](https://github.com/Restioson/xtra/blob/master/examples)
-to get started! Enabling the `tokio`, `async_std`, `smol`, or `wasm_bindgen` features
-is recommended in order to enable some  convenience methods (such as `xtra::spawn_tokio`). Which you enable will depend on
-which executor you want to use (check out their docs to learn more about each). If you have any questions, feel free to
-[open an issue](https://github.com/Restioson/xtra/issues/new) or message me on the [Rust discord](https://bit.ly/rust-community).
+
+Check out the [docs](https://docs.rs/xtra) and the [examples](https://github.com/Restioson/xtra/blob/master/examples) to get started!
+Enabling the `tokio`, `async_std`, `smol`, or `wasm_bindgen` features is recommended in order to enable some  convenience methods (such as `xtra::spawn_tokio`).
+Which you enable will depend on which executor you want to use (check out their docs to learn more about each).
+If you have any questions, feel free to [open an issue](https://github.com/Restioson/xtra/issues/new) or message me on the [Rust discord](https://bit.ly/rust-community).
```
Comment (thomaseizinger) [inline README.md:61]: I changed this to one sentence per line to make working with github's suggestion easier.

File: README.md (hunk 1 above)
Comment (Restioson) [reply README.md:8]: I think this is fine to be honest

**Review by Restioson — APPROVED** [review]

**Comment by Restioson** [comment]
Looks good, thanks!

**Pull request merged**